src/components/AppDialog/AppDialog.component.tsx

```diff
@@ -0,0 +1,58 @@
+import { IconButton, Typography } from '@mui/material';
```

Comment:
please use direct imports to minimialze bundle size:
e.g.
```
import IconButton from '@mui/material/IconButton';
```
https://mui.com/guides/minimizing-bundle-size/